### PR TITLE
Added recommendation to use Stetho for application debugging.

### PR DESCRIPTION
**Pull request by kypeli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -548,6 +549,10 @@ You still need to write some parsing code yourself to read the data objects from
 We generally do not recommend using an Object-Relation Mapping library unless you have unusually complex data and you have a dire need. They tend to be complex and require time to learn. If you decide to go with an ORM you should pay attention to whether or not it is _process safe_ if your application requires it, as many of the existing ORM solutions surprisingly are not.
 
 
+### Use Stetho 
+
+[Stetho](http://facebook.github.io/stetho/) is a debug bridge for Android applications from Facebook that integrates with the Chrome desktop browser's Developer Tools. With Stetho you can easily inspect your application, most notebly the network traffic if you use the OkHttp HTTP client. It also allows you to easily inspect SQLite databases and the shared preferences in your app.
```
Comment (peter-tackage) [inline README.md]: "notably", not "notebly".

**Comment by peter-tackage** [comment]
Good stuff, @kypeli . What do you think about adding a sentence stating that you should only use it for debug build variants and not release?

File: README.md (hunk 1 above)
Comment (kypeli) [inline README.md]: Thanks!

**Comment by kypeli** [comment]
I think that's a good point. Will add.

**Pull request merged**